pkg/kubelet/rkt/rkt.go

```diff
@@ -78,11 +78,11 @@ const (
 	k8sRktNameAnno         = "rkt.kubernetes.io/name"
 	k8sRktNamespaceAnno    = "rkt.kubernetes.io/namespace"
 	//TODO: remove the creation time annotation once this is closed: https://github.com/coreos/rkt/issues/1789
-	k8sRktCreationTimeAnno  = "rkt.kubernetes.io/created"
-	k8sRktContainerHashAnno = "rkt.kubernetes.io/containerhash"
-	k8sRktRestartCountAnno  = "rkt.kubernetes.io/restartcount"
-
-	dockerPrefix = "docker://"
+	k8sRktCreationTimeAnno     = "rkt.kubernetes.io/created"
+	k8sRktContainerHashAnno    = "rkt.kubernetes.io/containerhash"
+	k8sRktRestartCountAnno     = "rkt.kubernetes.io/restartcount"
+	k8sRktContainerLogPathAnno = "rkt.kubernetes.io/containerlogpath"
```

Comment:
nit: Termination message is different from the container log. Can we rename this to make it clear?